libraries/ms-common/src/x86/flash.c

```diff
@@ -0,0 +1,87 @@
+#include "flash.h"
```

Comment:
Why not use [mmap](http://man7.org/linux/man-pages/man2/mmap.2.html)? Not a issue just something to consider.

Reply:
Using a file allows the flash to persist across program executions. I'll probably add an environment variable so the user can set the location if they want.

Reply:
We should probably default the environment variable to something unique per `PROJECT`?

Reply:
Sure, makes sense.